lib/sphero.js

```diff
@@ -217,6 +271,16 @@ Sphero.prototype._execCallback = function(seq, packet) {
   }
 };
 
+/**
+ * Returns the response cmd (did, cid) passed to the sphero
+ * based on the seq from the response (used for parsing responses).
+ *
+ * @private
+ * @param {Number} seq from the sphero response packet
+ * @example
+ * sphero._responseCmd(0x14);
+ * @return {Object} containing cmd ids { did: number, cid: number }
```

Comment:
Type for this should be marked as `{Object|void}` to account for different possible return values.